{"filename": "python-pytest-rerunfailures-14.0-1-any.pkg.tar.zst", "name": "python-pytest-rerunfailures", "base": "python-pytest-rerunfailures", "version": "14.0-1", "desc": "A plugin for py.test that re-runs failed tests to eliminate intermittent failures", "csize": "25613", "isize": "113293", "md5sum": "65a806cafa0615069016bd02068b24ac", "sha256sum": "16df75ed55ec6832e9646dd303bdcf6358e6e9cbf13ced41c9af232d376f3378", "pgpsig": "iHUEABYKAB0WIQRizHP4hOUpV7L92IObeih9mi7GCAUCZfHvIgAKCRCbeih9mi7GCPXoAP9RrKRPNLN5DiwxCNOreq5kcSSk1cbOkKB+5ox0wjNFLAD/cf7uy+O/+MJrI2kH604vHpHM/iOUsmHMl+P1ZWY32QU=", "url": "https://github.com/pytest-dev/pytest-rerunfailures/", "license": "MPL-2.0", "arch": "any", "builddate": "1710354069", "packager": "David Runge <dvzrv@archlinux.org>", "depends": ["python", "python-packaging", "python-pytest"], "optdepends": "python-pytest-xdist: for recovering from crashes", "makedepends": ["python-build", "python-installer", "python-setuptools", "python-wheel"], "checkdepends": ["python-pytest-xdist"], "tokens": ["python-pytest-rerunfailures", "python-pytest-rerunfailures_x86_64", "python-pytest-rerunfailures_testing", "python-pytest-rerunfailures_x86_64_testing"]}